### ISSUE TITLE
Ansible version requirements

### ISSUE THREAD

**Issue by deimosfr** [UnderGreen/ansible-role-mongodb#6]
Hi,

I've got an issue when trying to launch your playbook with Ansible 1.7.2 in Debian Jessie package:
```
ansible-playbook -i hosts site.yml --limit localhost -vvvv
ERROR: mongodb_replication is not a legal parameter in an Ansible task or handler
```
I tested with pip, and it's the same. However, with 1.8.4 version, it works like a charm. Is there any way to backport the compatibility to 1.7 ?

Note: you should update meta role information

Thanks

**Comment by UnderGreen** [comment]
Hello.
Problem is in old behavior to search modules. Fixed in ansible 1.8.0: https://github.com/ansible/ansible/commit/e5116d2f9bd851949ae50e0c9c112750e7cec761
I found only one workaround: create symlink in library
```
ln -s mongodb_replication.py mongodb_replication
```
and play with -M roles/*/library:
```
ansible-playbook -i hosts all -M roles/*/library
```

**Comment by deimosfr** [comment]
Thanks !
